tests/baselines/reference/YieldStarExpression4_es6.types

```diff
@@ -2,11 +2,12 @@
 
 === YieldStarExpression4_es6.ts ===
 function *g() {
->g : () => Generator<any, void, undefined>
->  : ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+>g : () => Generator<any, void, unknown>

```

Comment:
Is it weird to think that it's almost better for the type for an untyped `yield` result to be `void` instead of `unknown`? I guess it'd be funky seeing the `void` in the parameter list of the `.next` call, but it feels like if you wrote `function foo*() { return yield val }`, having `foo`'s `TReturn` be `void` instead of `unknown` would be kinda nice.

I dunno, probably not terribly important.

Reply:
Having `TNext` default to either `void` or `undefined` ended up being an even larger breaking change.